src/components/Badge/badge.scss

```diff
@@ -8,69 +8,72 @@
     border-radius: 18px;
     padding: 0px 12px 0px 8px;
     width: fit-content;
-    svg{
-        margin-right: 8px;
+    span{
+        margin-left: 5px;
     }
+    // svg{
+    //     margin-right: 8px;
+    // }
 }
 
 .swb-live{
     background-color:hsla(123, 54%, 74%, 0.2);
     font-size:13px;
-    svg{
-        circle{
-            fill:#5EAB6C,
-        }
-    }
+    // svg{
+    //     circle{
+    //         fill:#5EAB6C,
+    //     }
+    // }
 }
 
 .swb-recording{
     background-color:#FFFFFF;
     font-size:11px;
-    svg{
-        circle{
-            fill:#943734;
-            animation: 1s blink ease infinite,
-        }
-    }
+    // svg{
```

Comment:
Could we get rid of the commented code, if we will no longer use it?

Reply:
Cleaned.